pkg/apis/cluster/common/consts.go

```diff
@@ -103,3 +103,59 @@ const (
 	// i.e gradually scale down the old MachineSet and scale up the new one.
 	RollingUpdateMachineDeploymentStrategyType MachineDeploymentStrategyType = "RollingUpdate"
 )
+
+// MachinePhase is the ongoing-phase machine is going through at the moment.
+type MachinePhase string
+
+// These are the valid values for the MachinePhase.
+const (
+	// MachinePending should be set when machine is being created but it has not joined the
+	// cluster yet.
+	MachinePending MachinePhase = "Pending"
+
+	// MachineRunning should be set when machine has joined the cluster and running successfully.
+	MachineRunning MachinePhase = "Running"
```

Comment:
This is the same as `Machine` exists and has a `.Status.NodeRef` that points to a healthy node